validator_client/src/http_api/tests.rs

```diff
@@ -1,5 +1,6 @@
 #![cfg(test)]
-#![cfg(not(debug_assertions))]
+// DO NOT MERGE
```

Comment:
I will remove this after the review. Rust analyzer does not recognize the test file with this option.